apps/desktop/src/components/views/terminal/terminal-panel.tsx

```
@@ -0,0 +1,430 @@
+      }
+    };
+
+    window.addEventListener('keydown', handleKeyDown);
+    return () => window.removeEventListener('keydown', handleKeyDown);
+  }, [zoomIn, zoomOut, resetZoom]);
+
+  return (
+    <div
+      className={cn('flex h-full flex-col', isActive && 'ring-1 ring-primary ring-inset')}
+      onClick={onFocus}
+    >
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)